packages/eslint-plugin-testing-library-react/docs/rules/prefer-expect-query-by.md

```diff
@@ -0,0 +1,28 @@
+# Avoid using `expect(getBy*)` (prefer-expect-query-by)
+
+React (DOM) Testing Library support two types of queries: `getBy*` and `queryBy*`. Using `getBy*` throws an error in case the element is not found. This is useful when using method like `waitForElement`, which are `async` functions and therefore throwing an error will cause the test to stop.
+However, when trying to assert if an element is not in the document, we obviously can't use `getBy*`. Instead we should use `queryBy*`, which does not throw and therefore we can assert that `expect(queryByText("Foo")).not.toBeInTheDocument()`.
```

Comment:
I would avoid words like "obviously" it can easily be perceived as "pointy" especially as this is Open Source.

"Instead it is recommended to use `getQueryBy*` to receive better error messages." or something.